descriptions/api.github.com/api.github.com.yaml

```diff
@@ -7984,6 +8005,17 @@ paths:
       - "$ref": "#/components/parameters/org"
       - "$ref": "#/components/parameters/per_page"
       - "$ref": "#/components/parameters/page"
+      - name: exclude
```

Comment:
Did you consider adding an exclude parameter globally to all endpoints? 

In this PR you introduce `exclude` and `excludedAttributes` (see below). Maybe use one name to keep it coherent?

Reply:
@gr2m unfortunately these are not new parameters, but rather ones we've identified as missing. I agree with you on consistency, some we keep an eye for going forward.